app/controllers/epp_controller.rb

```diff
@@ -407,4 +400,26 @@ def session_timeout_reached?
     timeout = 5.minutes
     epp_session.updated_at < (Time.zone.now - timeout)
   end
+
+  def iptables_counter_update
+    return if ENV['iptables_counter_enabled'].blank? && ENV['iptables_counter_enabled'] != 'true'
```

Comment:
The first part of this condition can be skipped, it's contained within the second one:

```ruby
2.3.7 :004 > ''.blank?
 => true
2.3.7 :005 > '' != 'true'
 => true
2.3.7 :006 > nil != 'true'
 => true
2.3.7 :009 > 'false' != 'true'
 => true
2.3.7 :008 > 'true' != 'true'
 => false
```